src/js/lib/safe.js

```diff
@@ -44,15 +36,21 @@ var makeTranslateStack = function(stackLineRegExp, translateLine) {
         line = lines[i] = translateLine.apply(this, m);
       }
       if (line.match(module.filename)) {
-        lines.splice(--i, 2);
+        lines.splice(i, 1);
```

Comment:
Agreed, there should be unit tests. This used to remove two lines that were useless, but that's tricky and not always the case. Now it removes the current line if it is safe.js, which is much more readable.